app.js

```diff
@@ -103,7 +103,7 @@ function onSearch(data) {
       // create card back
       // create card back text
       const labels = [
-        { label: 'Type Species', data: dinosaur.typeSpecies },
+        { label: 'Type', data: dinosaur.typeOfDinosaur},
```

Comment:
I had only just noticed that those two were the same, and you've fixed it already.